readme.md

```diff
@@ -172,7 +172,8 @@ useStore.destroy()
 
 // You can of course use the hook as you always would
 function Component() {
-  const paw = useStore(state => state.paw)
+  const paw = useStore((state) => state.paw)
+}
```

Comment:
```suggestion
  // ...
```
Wouldn't it better if we were to add one line?

Reply:
> Wouldn't it better if we were to add one line?

I was thinking about something, Is it necessary to add corresponding hints? (`(Other codes in the Component)`)
After adding it, although it looks quite obvious. But it may bring about the situation of "too much content, not concise". 🤔
```ts
// ... (Other codes in the `Component`)
```
or
```ts
// ...
```